src/Divider/Divider.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render } from '../test-utils';
+import { Divider } from './Divider';
+import { cleanup } from '../test-utils';
+
+describe('Divider', () => {
+  it('renders by default', async () => {
+    const { findByRole } = render(<Divider />);
+    expect(await findByRole('separator')).toBeInTheDocument();
+  });
+
+  it('render horizontally', async () => {
+    const { findByRole } = render(<Divider orientation="horizontal" />);
+    expect(await findByRole('separator')).toBeInTheDocument();
+  });
+
+  it('renders vertically', async () => {
+    const { findByTestId } = render(<Divider data-testid="content-divider" orientation="vertical" />);
+    expect(await findByTestId('content-divider')).toBeInTheDocument();
+  });
+
+  it('renders with all combintaions of props horizontal', async () => {
+    let component;
+
+    component = render(<Divider orientation="horizontal" size="small" />);
+    expect(await component.findByRole('separator')).toHaveStyle('height: 1px; width:auto');
```

Comment:
I think it's best not to use styles to make assertions: https://testing-library.com/docs/queries/about#priority